src/services/enum-utilities.ts

```diff
@@ -332,13 +334,15 @@ export default class EnumUtilities {
   }
 
   /**
-   * Converts a string in "camelCase" (or "PascalCase") to separate, title-case words,
+   * Converts a string in "camelCase" (or "PascalCase") to a string of separate, title-case words,
    * suitable for a title or proper name.
    * @param s the string to convert
    * @returns the converted string
    */
   static camelCaseToWords (s: string): string {
-    return s?.split(/(?=[A-Z])/).join(' ').replace(/^\w/, c => c.toUpperCase()) || ''
+    const words = s?.split(/(?=[A-Z])/).join(' ').replace(/^\w/, c => c.toUpperCase()) || ''
+    // SPECIAL CASE: convert 'Agm' to uppercase
+    return words.replace('Agm', 'AGM')
```

Comment:
The alternative is a large effort to include the document name in the API response (requiring API work and possibly informing third parties/delays) and then use that.

This is a special case, but it's a simple one, and we could add more here if needed, so I think it's not the best solution but it's a solution we can afford and it's functional and understandable and safe. Thoughts?

Reply:
I think it's OK for now Sev. If more and more arise in the future, then we can think about going the long and hard way.